cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,23 @@
-describe('', () => {
-  before(() => {
-
-  });
-
-  it('', () => {
-
-  });
-
-  it('', () => {
+describe('Create and Delete Article Test', () => {
+  const uniqueUsername = `user${Math.floor(Math.random() * 100000)}`;
+  const uniqueEmail = `${uniqueUsername}@example.com`;
+  const uniquePassword = 'password123';
```

Comment:
For the future: everything that is repeated in the code more than once can be moved to a separate variable or function. You use a randomizer in several constants. Why don't you also put it in a separate constant?